### DISCUSSION TITLE
Self-contained wheels with no dependency on `cargo` for installation

### DISCUSSION THREAD

**Discussion by cswinter** [PyO3/maturin#723] [Q&A] [answered]
I have a Python package published to PyPI with maturin: https://github.com/entity-neural-network/ragged-buffer
Users are running into errors about `cargo` missing when installing the package. Is there a way to publish a package that does not have any install or runtime dependencies on `cargo`?

**Comment by messense** [comment, marked as answer]
I see you already published binary wheels for that package, so the users running into errors about `cargo` missing may use an outdated `pip`. Upgrading `pip` should resolve the issue.
